prow/plugins/updateconfig/updateconfig.go

```diff
@@ -240,6 +240,22 @@ func FilterChanges(cfg plugins.ConfigUpdater, changes []github.PullRequestChange
 			}
 		}
 	}
+	return handleDefaultNamespace(toUpdate, defaultNamespace)
+}
+
+// handleDefaultNamespace ensures plugins.ConfigMapID.Namespace is not empty string
+func handleDefaultNamespace(toUpdateBefore map[plugins.ConfigMapID][]ConfigMapUpdate, defaultNamespace string) map[plugins.ConfigMapID][]ConfigMapUpdate {
+	toUpdate := map[plugins.ConfigMapID][]ConfigMapUpdate{}
```

Comment:
Why do we need to duplicate the structure?

Reply:
Done.